con-base/src/test/java/ba/codecentric/base/validation/FacilityTest.java

```diff
@@ -19,7 +18,6 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
-@SpringBootConfiguration
 public class FacilityTest {
```

Comment:
You don't need any of these annotations. And tests are still failing. Tests needs to be executed before pushing to branch and looks like your are not doing that ;)